inventory/byo/hosts.example

```diff
@@ -811,8 +811,11 @@ openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true',
 # Or you may optionally define your own build overrides configuration serialized as json
 #openshift_buildoverrides_json='{"BuildOverrides":{"configuration":{"apiVersion":"v1","kind":"BuildDefaultsConfig","forcePull":"true"}}}'
 
-# Enable template service broker by specifying one of more namespaces whose
-# templates will be served by the broker
+# Enable template service broker
+#openshift_enable_service_catalog=true
```

Comment:
should this be somewhere else, or at least under its own comment?

Reply:
(In theory enabling the SC and enabling the TSB are distinct choices, though obviously enabling the TSB w/o enabilng the SC would be a bad choice. but the other way around certainly makes sense)

Reply:
currently ansible does nothing if template_service_broker_install == true && openshift_enable_service_catalog != true - should I be more explicit about this in the comment?

Reply:
yes i think so